cime_config/config_component.xml

```diff
@@ -295,6 +297,26 @@
       ABBY,BLAN,CPER,DEJU,GRSM,HEAL,KONA,LENO,NIWO,ONAQ,PUUM,SERC,SRER,TALL,TREE,WOOD,
       BARR,BONA,DCFS,DELA,GUAN,JERC,KONZ,MLBS,NOGP,ORNL,RMNP,SJER,STEI,TEAK,UKFS,WREF,
       BART,CLBJ,DSNY,HARV,JORN,LAJA,MOAB,OAES,OSBS,SCBI,SOAP,STER,TOOL,UNDE,YELL,
+      NEON_PRECIP.ABBY,NEON_PRECIP.BLAN,NEON_PRECIP.CPER,NEON_PRECIP.DEJU,NEON_PRECIP.GRSM,
+      NEON_PRECIP.HEAL,NEON_PRECIP.KONA,NEON_PRECIP.LENO,NEON_PRECIP.NIWO,NEON_PRECIP.ONAQ,
+      NEON_PRECIP.PUUM,NEON_PRECIP.SERC,NEON_PRECIP.SRER,NEON_PRECIP.TALL,NEON_PRECIP.TREE,
+      NEON_PRECIP.WOOD,NEON_PRECIP.BARR,NEON_PRECIP.BONA,NEON_PRECIP.DCFS,NEON_PRECIP.DELA,
+      NEON_PRECIP.GUAN,NEON_PRECIP.JERC,NEON_PRECIP.KONZ,NEON_PRECIP.MLBS,NEON_PRECIP.NOGP,
+      NEON_PRECIP.ORNL,NEON_PRECIP.RMNP,NEON_PRECIP.SJER,NEON_PRECIP.STEI,NEON_PRECIP.TEAK,
+      NEON_PRECIP.UKFS,NEON_PRECIP.WREF,NEON_PRECIP.BART,NEON_PRECIP.CLBJ,NEON_PRECIP.DSNY,
+      NEON_PRECIP.HARV,NEON_PRECIP.JORN,NEON_PRECIP.LAJA,NEON_PRECIP.MOAB,NEON_PRECIP.OAES,
+      NEON_PRECIP.OSBS,NEON_PRECIP.SCBI,NEON_PRECIP.SOAP,NEON_PRECIP.STER,NEON_PRECIP.TOOL,
+      NEON_PRECIP.UNDE,NEON_PRECIP.YELL,
+      PRISM_PRECIP.ABBY,PRISM_PRECIP.BLAN,PRISM_PRECIP.CPER,PRISM_PRECIP.DEJU,PRISM_PRECIP.GRSM,
+      PRISM_PRECIP.HEAL,PRISM_PRECIP.KONA,PRISM_PRECIP.LENO,PRISM_PRECIP.NIWO,PRISM_PRECIP.ONAQ,
+      PRISM_PRECIP.PUUM,PRISM_PRECIP.SERC,PRISM_PRECIP.SRER,PRISM_PRECIP.TALL,PRISM_PRECIP.TREE,
+      PRISM_PRECIP.WOOD,PRISM_PRECIP.BARR,PRISM_PRECIP.BONA,PRISM_PRECIP.DCFS,PRISM_PRECIP.DELA,
+      PRISM_PRECIP.GUAN,PRISM_PRECIP.JERC,PRISM_PRECIP.KONZ,PRISM_PRECIP.MLBS,PRISM_PRECIP.NOGP,
+      PRISM_PRECIP.ORNL,PRISM_PRECIP.RMNP,PRISM_PRECIP.SJER,PRISM_PRECIP.STEI,PRISM_PRECIP.TEAK,
+      PRISM_PRECIP.UKFS,PRISM_PRECIP.WREF,PRISM_PRECIP.BART,PRISM_PRECIP.CLBJ,PRISM_PRECIP.DSNY,
+      PRISM_PRECIP.HARV,PRISM_PRECIP.JORN,PRISM_PRECIP.LAJA,PRISM_PRECIP.MOAB,PRISM_PRECIP.OAES,
+      PRISM_PRECIP.OSBS,PRISM_PRECIP.SCBI,PRISM_PRECIP.SOAP,PRISM_PRECIP.STER,PRISM_PRECIP.TOOL,
+      PRISM_PRECIP.UNDE,PRISM_PRECIP.YELL,
```

Comment:
Since we don't have PRISM data from sites outside CONUS (e.g. TOOL, BONA, LAJA, etc) should we have these even be accepted values for CLM_USRDAT_DIR?  I'm included to say no, so that requests for PRISM_PRECIP.HEAL would fail?  

That said, keeping them would provide a placeholder so that users could create PRECIP data from locations where we're missing PRISM data more easily.